### ISSUE TITLE
Spaces in PATH breaks $(ncrystal-config --setup)

### ISSUE THREAD

**Issue by tkittel** [mctools/ncrystal#52]
`$(ncrystal-config --setup)` (potentially through `setup.sh`) does not work when `PATH` contains spaces. This was observed by @nicriz during plugin development on Windows WSL:

```
-bash: export: `'PATH=/home/nicorizzi/ncrystal_install/bin:/home/nicorizzi/phits/dchain-sp/bin:/home/nicorizzi/phits/bin:/home/nicorizzi/anaconda3/bin:/home/nicorizzi/anaconda3/condabin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/mnt/c/Python27/Lib/site-packages/PyQt4:/mnt/c/Program': not a valid identifier
-bash: export: `Files/Python38/Scripts:/mnt/c/Program': not a valid identifier
-bash: export: `Files/Python38:/mnt/c/WINDOWS/system32:/mnt/c/WINDOWS:/mnt/c/WINDOWS/System32/Wbem:/mnt/c/WINDOWS/System32/WindowsPowerShell/v1.0:/mnt/c/WINDOWS/System32/OpenSSH:/mnt/c/Python27:/mnt/c/Python27/DLLs:/mnt/c/Python27/Scripts:/mnt/c/Python27/Lib/site-packages/vtk:/mnt/c/Python27/gnuplot/binary:/mnt/c/Program': not a valid identifier
-bash: export: `(x86)/pythonxy/SciTE-3.2.2-1:/mnt/c/Program': not a valid identifier
-bash: export: `(x86)/pythonxy/console:/mnt/c/MinGW32-xy/bin:/mnt/c/Program': not a valid identifier
-bash: export: `(x86)/MiKTeX': not a valid identifier
-bash: export: `2.9/miktex/bin:/mnt/c/Users/nicri/AppData/Local/Microsoft/WindowsApps:/mnt/c/Users/nicri/AppData/Local/Programs/Microsoft': not a valid identifier
-bash: export: `Code/bin:/snap/bin'': not a valid identifier

As you can see, it breaks the line everytime there's a space, like in Program Files in the first two lines
```

Which I can reproduce on Ubuntu/bash with a simple:
```
(tkittel@tkittel-XPS-13-9380 ~)> $(echo "export 'LALA=a b c'")
bash: export: `'LALA=a': not a valid identifier
bash: export: `c'': not a valid identifier
```

It is unclear to me how to do the escaping properly for it to work in this case (I had assumed that Python's `shlex.quote` would just deal with it), so it will need a bit of investigation to find a good fix.

**Comment by tkittel** [comment]
@nicriz could you try if it works better for you if instead of typing `$(ncrystal-config --setup)` you type:
```
 . <(ncrystal-config --setup)
```

**Comment by tkittel** [comment]
Or:
```
eval `ncrystal-config --setup`
```

**Comment by tkittel** [comment]
Or:
```
eval $(ncrystal-config --setup)
```

**Comment by nicriz** [comment]
Hi, all of three work smoothly for me :)

**Comment by tkittel** [comment]
Great, thanks for verifying! I'll do a few checks on non-BASH shells as well.

**Comment by tkittel** [comment]
So I'll adopt the `eval $(...)` version in the next NCrystal release. As a workaround until then, you can simply hand-edit the setup.sh and unsetup.sh files in your NCrystal installation and perform the change:
```diff
-    $( "$tmpnccfg" --setup )
+    eval $( "$tmpnccfg" --setup )
```
And likewise in unsetup.sh. This should fix Bash4 (most linux dists + WSL), Bash3 (old OSX) and zsh (new OSX). It still won't work for tcsh (BSD) or dash (ubuntu default). For that I have opened #54 for another time.

**Comment by tkittel** [comment]
So the change is now published in the new NCrystal release, v2.3.0.
